lib/bolt/util.rb

```diff
@@ -202,6 +202,29 @@ def validate_file(type, path, allow_dir = false)
         raise Bolt::FileError.new("The #{type} '#{path}' does not exist", path)
       end
 
+      # Check if there is a case-insensitive match to the path
+      def check_path_case(type, paths)
+        return if paths.nil?
+        matches = matching_paths(paths)
+        warn_path_case(type, matches) if matches.any?
+      end
+
+      def matching_paths(paths)
```

Comment:
This can also probably just be in check_path_case, unless there's somewhere else you want to use it?

Reply:
I had that originally, but appveyor kept failing the test I wrote when it was originally expecting output, so I changed the test to expect possible paths instead. I'd like to get it into `warn_path_case` if I can, but am not sure how to get the test working with my original test.